src/JLSOFile.jl

```diff
@@ -79,13 +95,29 @@ function Base.:(==)(a::JLSOFile, b::JLSOFile)
         a.version == b.version &&
         a.julia == b.julia &&
         a.image == b.image &&
-        a.pkgs == b.pkgs &&
+        a.manifest_toml == b.manifest_toml &&
```

Comment:
Is it going to be a problem that you can have 2 equiv Manifest.tomls
that differ in order but are the same?
Probably not because they are always alphabeticalized by Pkg.

Reply:
> Probably not because they are always alphabeticalized by Pkg.

Correct. This was actually the initial reason for considering saving the parsed toml, but yeah, Pkg.jl should consistently sort the packages.